docs/topics/db/transactions.txt

```diff
@@ -238,12 +238,6 @@ Django provides a single API to control database transactions.
     is especially important if you're using :func:`atomic` in long-running
     processes, outside of Django's request / response cycle.
 
-.. warning::
```

Comment:
If we want a `versionchanged`, this could work:

```
.. versionchanged:: 4.1
    
    Atomic blocks under :class:`django.test.TestCase` now check for durability
    so there’s no need to switch to :class:`django.test.TransactionTestCase` to
    test such usage.
```

Reply:
It might be useful to add some docs to explain what is going on.

From what I understand, the test runner will detect whether or not the atomic block is the one started by the TestCase, or a different atomic block. (So the disabling of the check is more sophisticated now.) If we don't put this in the public docs, at least a comment in the source code might be helpful.